yt_dlp/extractor/generic.py

```diff
@@ -2521,6 +2521,7 @@ def _real_extract(self, url):
                     doc,
                     mpd_base_url=full_response.url.rpartition('/')[0],
                     mpd_url=url)
+                info_dict['is_live'] = doc.get('type') == 'dynamic'
```

Comment:
since we set `live_status` elsewhere in the extractor, let's use that instead of `is_live` for consistency
```suggestion
                info_dict['live_status'] = 'is_live' if doc.get('type') == 'dynamic' else None
```

Reply:
```
    is_live:        True, False, or None (=unknown). Whether this video is a
                    live stream that goes on instead of a fixed-length video.
    was_live:       True, False, or None (=unknown). Whether this video was
                    originally a live stream.
    live_status:    None (=unknown), 'is_live', 'is_upcoming', 'was_live', 'not_live',
                    or 'post_live' (was live, but VOD is not yet processed)
                    If absent, automatically set from is_live, was_live
```
is_live and live_status  don't serve  the same purpose